Mostrar a janela inteira com Glade

JPTIZ opened the pull request:
- Arquivos .ui foram movidos para a pasta `res/ui`;
- A janela agora é construída a partir do `mainwindow.ui`, e não diretamente em C++;
- O código de `rudolph::gui::MainWindow` foi descartado (dá para usar diretamente o .ui por enquanto).

JPTIZ committed:
Move .ui files into res/ui.

JPTIZ committed:
Building window from glade.

JPTIZ committed:
Change main_window element class from Window to ApplicationWindow.

JPTIZ committed:
Replace rudolph::gui::MainWindow with Gtk::ApplicationWindow.

thurbridi reviewed (dismissed):
Elementos fora de alinhamento ao dar resize na MainWindow

![review](https://user-images.githubusercontent.com/12735856/54358595-8765f200-463f-11e9-8957-a3e5f9c9e404.png)

JPTIZ committed:
Fix some mainwindow.ui structural issues.

The main problem is that, on resize, the window size and command line
editboxes do not resize proportionaly (specially on shrink). This commit
changes, along with other small things, the UI hierarchy by separating
the main window into a 1x2 (rows x columns) grid, and the left panel is
a 2x1 subgrid (one for Display File management and other for
Navigation).

thurbridi committed:
MainWindow rework
